### PR TITLE
tidy up ur code :))

### PR DESCRIPTION
**Pull request by SmallPP420**

### CODE REVIEW

File: src/Program.cs [hunk 1]
```diff
@@ -11,34 +11,38 @@ class Program
...
+            foreach (var argument in splittedString)
             {
-                if (argument.Contains("--latency")) {
-                    string[] msec = argument.Split('=');
+                switch(argument)
+                {
+                    when argument.Contains("latency=")
+                        string[] msec = argument.Split('=');
                     string wordDelay = msec[1];
                     if(!int.TryParse(wordDelay, out delay)) {
```
Comment (usrDottik) [inline src/Program.cs:24]: if never closes

**Review by SmallPP420 — COMMENTED** [review]
oops i mustve copied the wrong code (i use messagelogger on bd so it tracked my edits)

File: src/Program.cs (hunk 1 above)
Comment (usrDottik) [reply src/Program.cs:24]: yeah, but you didn't fix the if being open :)

**Comment by SmallPP420** [comment]
poop

**Review by usrDottik — COMMENTED** [review]
Thicc

**Pull request merged**